lib/Configuration.js

```diff
@@ -64,6 +65,7 @@ const DEFAULT_CONFIG = {
   stripFileExtensions: ['.js', '.jsx'],
   tab: '  ',
   useRelativePaths: true,
+  whitespace: 'strip',
```

Comment:
I'm a little hesitant to introduce a bunch of extra config options. Is there any compelling reason to not just do this for everyone?